### PR TITLE
OCP4/CIS: kubelet_configure_event_creation e2e remediation

### PR DESCRIPTION
**Pull request by mrogers950**

### CODE REVIEW

File: applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
 ---
-default_result: FAIL
+default_result:
+  worker: FAIL
+  master: FAIL
+result_after_remediation:
+  worker: PASS
+  master: FAIL
```
Comment (JAORMX) [inline applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml:7]: Is this remediation not relevant on the master nodes?

Comment (mrogers950) [reply applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml:7]: It's relevant, but I kept it to the worker nodes to cut down on time.

**Pull request merged**